apps/i18n/spritelab/en_us.json

```diff
@@ -119,21 +119,27 @@
   "cscAdaptationsUpDownArrowKey": "Make sure to press the Up and Down arrow keys after pressing *\"Run\"*. You can click the orange buttons under the play area or press the keys on your keyboard.",
   "cscAdaptationsWhenRightPressed": "Make sure to add the  <code style=\"color:black; background-color:#6AF36C\">when right pressed</code> block with a new behavior for the sprite.",
   "cscAdaptationsWhenTouchEvent": "Make sure to add a `when touches` block for your sprites that will touch.\n<xml>\n<block type=\"gamelab_checkTouching\">\n  <field name=\"CONDITION\">\"when\"</field>\n  <value name=\"SPRITE1\">\n    <block type=\"gamelab_allSpritesWithAnimation\" can_disconnect_from_parent=\"false\">\n      <field name=\"ANIMATION\">\"octopus_red\"</field>\n    </block>\n  </value>\n  <value name=\"SPRITE2\">\n    <block type=\"gamelab_allSpritesWithAnimation\" can_disconnect_from_parent=\"false\">\n      <field name=\"ANIMATION\">\"underseadeco_34\"</field>\n    </block>\n  </value>\n</block>\n</xml>",
+  "cscBookcoverAddEventBlock": "Add a 'when up arrow clicked' event block",
+  "cscBookcoverAddOnclickBlock": "Be sure to add an event block for when the sprite is clicked.",
```

Comment:
@katiejofr Could/should this be `cscBookcoverAddOnClickBlock`? I see that the key as written matches what's in levelbuilder, but there's a comment that says, "You need to add a onClick block to your code," which matches what I expect. 

Also, I see some `TODO: add validation string:` comments in the `.level` files. Should we clean them up?

Reply:
I think this suggestion means an engineer would go in and update the raw .level files for these changes, which is something I'm hesitant about just from a process perspective - levelbuilder is very tricky when multiple people make edits to the same file, so trying to understand engineering edits vs levelbuilder edits and making sure we don't lose work from overwriting each other feels complex at this point.

The key needs to match in both places. So if the levels already have the lowercase camel case version, the priority now would be to keep them the same. I'd recommend Katie makes a ticket for the curriculum backlog to look at the next time we update these validation strings.

Also, because it's impossible to test feedback strings until they're in production, I wouldn't be surprised if it's already on the radar for someone will go back later to remove those TODOs once the strings have been tested in production. Like - for feedback strings, posting to production even when we're still testing and iterating is (unfortunately) part of the process, so I think it's more common for there to be these loose ends as we're still iterating.

Reply:
Dan, nailed it. Yes, I am waiting for these strings to be on production and have several levels bookmarked. I would love a way to easily find all of those "TODO:'s" so that I could loop back around and check into see what the contractor left and make sure I return to all of them. But in the meantime, I'm hesitant as Dan is. 

Thanks for jumping on this @ebeastlake!

Reply:
Sounds good to me. Thanks for the context!